packages/spor-react/src/theme/slot-recipes/tabs.ts

```diff
@@ -46,6 +75,15 @@ export const tabsSlotRecipe = defineSlotRecipe({
           },
           _active: {
             ...coreBackground("active"),
+            ...coreText("active"),
+          },
```

Comment:
Dette blir ikke helt riktig, ville skrevet det slik isteden:
```suggestion
          _active: {
            background: "core.surface.active",
            color: "text",
            outline: "none",
          },
```

Reply:
fortsatt litt usikker på dette, ta en titt på det om det er riktig nå